GameOfThrones/Migrations/202109141059199_AddCharacter.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace GameOfThrones.Migrations
+{
+    using System;
+    using System.Data.Entity.Migrations;
+    
+    public partial class AddCharacter : DbMigration
+    {
+        public override void Up()
+        {
+            CreateTable(
+                "dbo.Characters",
+                c => new
+                    {
+                        CharacterId = c.Int(nullable: false, identity: true),
+                        FirstName = c.String(),
+                        LastName = c.String(),
+                        Alive = c.Boolean(nullable: false),
+                        HouseId_HouseId = c.Int(),
+                    })
+                .PrimaryKey(t => t.CharacterId)
+                .ForeignKey("dbo.Houses", t => t.HouseId_HouseId)
+                .Index(t => t.HouseId_HouseId);
+
+            Sql("INSERT INTO Characters (FirstName, LastName, Alive, HouseId_HouseId) VALUES ('Jon', 'Snow', 1, 3)," +
```

Comment:
Interesting choice to run a manual SQL script as part of the migration. While it is something that you can do, it may have been best to add this to seed method or a separate `ContextInitialiser`

Reply:
I think it would've been better to put it in the seed method too but the first bullet point of step 4 said to

> populate the table by updating the generated migration

I'll have a look into ContextInitialisers to see what they do